org.civicrm.reqangular/src/common/angular-date/src/filters/CustomDateFilter.js

```diff
@@ -1,8 +1,10 @@
-module.exports = function ($filter, DateFactory) {
-    return function (datetime) {
+module.exports = function ($filter, DateFactory, DateFormatFactory) {
+    var dateFormat = null;
+    var filter = function (datetime, format) {
```

Comment:
So how did the usage of this filter change? You can now pass the format to the filter itself?